src/sentry/static/sentry/app/components/activity/note.jsx

```diff
@@ -6,7 +6,52 @@ import ConfigStore from '../../stores/configStore';
 import LinkWithConfirmation from '../../components/linkWithConfirmation';
 import {t} from '../../locale';
 
+function isSafeHref(href) {
+  try {
+    return /^https?:\/\/./i.test(decodeURIComponent(unescape(href)));
+  } catch(e) {
+    return false;
+  }
+}
+
+
+// We need to implement our own marked Renderer to not render
+// potentially malicious uris.
+// This is copy/pasted from
+// https://github.com/chjj/marked/blob/master/lib/marked.js#L869-L888
+// and modified.
+function Renderer(options) {
+  this.options = options || {};
+}
+
+Object.assign(Renderer.prototype, marked.Renderer.prototype);
+
+Renderer.prototype.link = function(href, title, text) {
+  // For a bad link, just return the plain text href
+  if (this.options.sanitize && !isSafeHref(href)) return href;
```

Comment:
> but it might confuse somebody pretty easily since you're not going to write a how-to anywhere

I think there are _more_ cases of something working unexpectedly by using an unintentional relative link, rather than people intentionally trying to use a relative url.

For example, doing something like `[foo](google.com)` is going to be relative.

I mean, I can make it work with relative urls, I just made an explicit decision against them because they seem more often to be erronous. I can't personally image _wanting_ to use a relative url in a comment.